src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.ConstantVelocity.cs

```diff
@@ -0,0 +1,146 @@
+﻿#if NET461
+#pragma warning disable CS0067
+#endif
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Uno.Disposables;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Windows.Devices.Input;
+using Uno.UI;
+using Uno.UI.DataBinding;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Input;
+using Uno;
+using Uno.Extensions;
+using Microsoft.Extensions.Logging;
+
+#if __ANDROID__
+using View = Android.Views.View;
+using Font = Android.Graphics.Typeface;
+#elif __IOS__
+using UIKit;
+using View = UIKit.UIView;
+using Color = UIKit.UIColor;
+using Font = UIKit.UIFont;
+#elif __MACOS__
+using View = AppKit.NSView;
+using AppKit;
+#else
+using View = Windows.UI.Xaml.UIElement;
+#endif
+
+#if UNO_HAS_MANAGED_SCROLL_PRESENTER
+using _ScrollContentPresenter = Windows.UI.Xaml.Controls.ScrollContentPresenter;
+#else
+using _ScrollContentPresenter = Windows.UI.Xaml.Controls.IScrollContentPresenter;
+#endif
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class ScrollViewer
+	{
+
+		private ConstantVelocityScroller? _constantVelocityScroller;
+		internal void SetConstantVelocities(float horizontalVelocity, float verticalVelocity)
+		{
+			_constantVelocityScroller ??= new ConstantVelocityScroller(this);
+			// Flip sign here for consistency with imported WinUI code
+			_constantVelocityScroller.HorizontalVelocity = -horizontalVelocity;
+			_constantVelocityScroller.VerticalVelocity = -verticalVelocity;
+		}
+
+		/// <summary>
+		/// Apply a constant scroll velocity, used as a substitute for constant-pan Manipulation functionality which Uno doesn't currently support.
+		/// </summary>
+		private class ConstantVelocityScroller
+		{
+			private readonly DispatcherTimer _timer = new DispatcherTimer();
```

Comment:
Recently I try to use only the `DispatcherQueueTimer`, looks more future proof.

Reply:
Good to know, I'll leave it like this for now.